draft-ietf-add-split-horizon-authority.xml

```diff
@@ -350,6 +361,28 @@
     </section>
     <!-- Validating -->
 
+    <section>
+      <name>Delegating DNSSEC across Split DNS Boundaries</name>
+      <t>We wish to enable DNSSEC validation of local DNS names without requiring
+      the local resolver to hold DNSSEC private keys that are valid for the parent
+      zone.  To support this configuration, parent zones <bcp14>MAY</bcp14> add a
```

Comment:
If the "ds" key is not added, DNSSEC validation by the client will be rejected. we should strongly recommend to add the "ds" key.